### DISCUSSION TITLE
game links? they got blovked

### DISCUSSION THREAD

**Discussion by ineedgamelinkstheygetblockedfast** [UseInterstellar/Interstellar#371] [General]
any unblocked game links

**Comment by 0mhx** [comment]
Skid

**Reply by ineedgamelinkstheygetblockedfast** [reply]
huh? whats the link

**Comment by 0mhx** [comment]
Stop f*cking asking for links here. Ask in the [Discord](https://discord.gg/interstellar) or you will be blocked from asking questions in the GitHub.

**Comment by xbubbo** [comment]
# Join our [Discord ](https://discord.gg/interstellar) server or host your own.
